src/components/dataset/query/sidebar/QueryViewSidebar.jsx

```diff
@@ -189,6 +184,15 @@ export class QueryViewSidebar extends React.PureComponent {
     this.setState({ searchString: event.target.value });
   };
 
+  handleOpenFilter = filter => {
+    const { openFilter } = this.state;
+    if (filter === openFilter) {
+      this.setState({ openFilter: '' });
```

Comment:
maybe look into changing openFilter to an object instead of a string so it doesn't persist across tables